dask/bag/random.py

```diff
@@ -21,11 +21,10 @@ def sample(population, k):
 
     Examples
     --------
-    >>> import dask.bag as db # doctest: +SKIP
-    ... from dask.bag import random
-    ...
-    ... b = db.from_sequence(range(5), npartitions=2)
-    ... list(random.sample(b, 3).compute())
+    >>> import dask.bag as db
+    >>> from dask.bag import random
+    >>> b = db.from_sequence(range(5), npartitions=2)
+    >>> list(random.sample(b, 3).compute())  # doctest: +SKIP
```

Comment:
Moving the SKIP to the _actual_ line that we want to skip feels more intuitive in my opinion.